docs/application/native/guides/device/system.md

```diff
@@ -631,6 +632,15 @@ The following table lists the speech feature keys:
 | `http://tizen.org/feature/speech.recognition` | `bool` | The platform returns `true` for this key, if the device supports speech recognition (STT). |
 | `http://tizen.org/feature/speech.synthesis` | `bool` | The platform returns `true` for this key, if the device supports speech synthesis (TTS). |
 
+The following table lists the external storage feature keys:
+
+<a name="storage"></a>
+**Table: External storage feature keys**
+
+| Key                                 | Type   | Description                              |
+|-------------------------------------|--------|------------------------------------------|
+| `http://tizen.org/feature/storage.external` | `bool` | The platform returns `true` for this key, if the device supports the external storage. |
```

Comment:
Please make this small change:

The platform returns `true` for this key, if the device supports external storage.

Reply:
Done.